src/pybind/mgr/cephadm/tests/test_cephadm.py

```diff
@@ -2493,6 +2493,34 @@ def test_tuned_profiles_validation(self, cephadm_module):
                     return_value=[mock.Mock().hostname, mock.Mock().hostname])
                 cephadm_module._validate_tuned_profile_spec(spec)
 
+    def test_set_cephadm_command_timeout(self, cephadm_module):
+        # Note that the value we pass to --timeout in the binary
+        # (which is what this func is for) subtracts 5 seconds so
+        # the timeout is slightly below the one used for the
+        # asyncio timeout in the mgr module. So we expect to get
+        # the value passed minus 5 in normal cases
```

Comment:
If we do the `-5` outside then there's no need for handling this small tweak not propagating it to the UT.